pkg/app/mssql_utils.go

```diff
@@ -0,0 +1,107 @@
+package app
+
+import (
+	"github.com/ghodss/yaml"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (m *MssqlDB) getDeploymentObj() (*appsv1.Deployment, error) {
```

Comment:
These methods - since they are implemented on the `MssqlDB` - can be kept in the `mssql.go` only. @viveksinghggits wdyt?